.gitignore

```diff
@@ -1,3 +1,5 @@
 _USER_DEFINES.h
 *backup*
 *aljaz*
+\.vscode
```

Comment:
Good practice is to put this into your private `.gitignore` file.

Reply:
The only point here is that you can say, it is in the "wrong" .gitignore file. It is in the "project" wide, in the root. If you watch the .gitignore in the "EleksTubeHAX_pio" folder, you see, that other files in/from the .vscode folder are ignored already. So this is just a generalization of the exsting ignores. Now, all files in any .vscode dir will be ignored.
BUT it has nothing to do with the IPSTUBE, so I will remove it.

Reply:
https://docs.github.com/en/get-started/getting-started-with-git/ignoring-files#configuring-ignored-files-for-all-repositories-on-your-computer

Reply:
https://medium.com/@peter_graham/how-to-create-a-local-gitignore-1b19f083492b